appveyor.yml

```diff
@@ -6,12 +6,22 @@ before_build:
   - nuget restore nunit.sln
 
 build_script: 
-  - msbuild nunit.sln /p:Configuration=Release /t:Rebuild
+  - ps: |
+      .\appveyor.ps1
```

Comment:
That makes sense, especially as I am allergic to both powershell and (even more) programming in XML. :-)